ramls/samples/profiles/mappingProfile.sample

```diff
@@ -9,7 +9,7 @@
   "transformations": [
     {
       "fieldId": "electronicAccess.linkText",
-      "path": "$.electronicAccess[*].linkText",
+      "path": "Json path for electronicAccess[*].linkText",
```

Comment:
Can you just add a comment above with the correct format, 
given that this is not the right format

Reply:
looks like there is no possibility to add a comment to the json